### PR TITLE
Create Blog app with a post model

### PR DESCRIPTION
**Pull request by RootenberG**

### CODE REVIEW

File: blog/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+"""Django website."""
```
Comment (webknjaz) [inline blog/__init__.py]:
```suggestion
"""Blog web app."""
```

File: blog/admin.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+"""Django admin coonfig for mysite."""
```
Comment (webknjaz) [inline blog/admin.py]: mysite?

File: blog/apps.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+"""Registry of installed apps for mysite."""
```
Comment (webknjaz) [inline blog/apps.py]: mysite?

File: blog/models.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
+"""Mysite models."""
```
Comment (webknjaz) [inline blog/models.py]: mysite? blog db models?

File: blog/models.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    created_date = models.DateTimeField(default=timezone.now)
+    published_date = models.DateTimeField(blank=True, null=True)
+
+    def publish(self):
+        """Mark the post as published."""
+        self.published_date = timezone.now()
+        self.save()
+
+    def __str__(self):
+        """Return post wich converted to string."""
```
Comment (webknjaz) [inline blog/models.py]: 1) "w***h***ich" is misspelled. the verb is missing, mind your grammar.
2) Plz read the whole data model doc page but especially [this paragraph](https://docs.python.org/3/reference/datamodel.html#object.__str__)
3) You should describe the effect of having this method here which is just support for rendering `Post` instances into strings (by passing it to the `str()` function, usually), it's a part of the Python's string conversion protocol.

So you should just say that this returns a string representation of a Post object. Or, as I usually say, it renders Post as a string.

File: blog/tests.py [hunk 6]
```diff
@@ -0,0 +1,4 @@
+"""Tests for mysite."""
```
Comment (webknjaz) [inline blog/tests.py]: blog

File: blog/views.py [hunk 7]
```diff
@@ -0,0 +1,4 @@
+"""Wiews for Blog app."""
```
Comment (webknjaz) [inline blog/views.py]:
```suggestion
"""Views for Blog app."""
```

**Comment by webknjaz** [comment]
"and" in the PR and commit message suggests that there's two separate changes meaning that they should go to separate atomic commits.

**Review by webknjaz — CHANGES_REQUESTED** [review]
Blocked on a non-atomic commit.

**Comment by webknjaz** [comment]
> Add migrations module

it's a package, not a module. + the first commit message still refers to multiple things

**Comment by RootenberG** [comment]
> > Add migrations module
> 
> it's a package, not a module. + the first commit message still refers to multiple things

Вo you mean a model creating?

**Comment by webknjaz** [comment]
I mean the commit message.

**Comment by RootenberG** [comment]
> I mean the commit message.


What changes should I share?

**Comment by webknjaz** [comment]
You shouldn't say that you exclude a module because you exclude a collection of modules which is a package.

**Comment by RootenberG** [comment]
> You shouldn't say that you exclude a module because you exclude a collection of modules which is a package.

I fixed it.

**Review by webknjaz — APPROVED** [review]
Perfect!

**Pull request merged**